app/src/components/fileTypes.js

```diff
@@ -0,0 +1,28 @@
+const log = require('npmlog');
```

Comment:
Doens't look like log is used in this script? Does lint catch the unused variable?

Reply:
Lint should be catching it - and if it isn't, I know vscode should be highlighting it if you have the eslint plugin installed.